test/build/55_cache.bats

```diff
@@ -1404,6 +1404,13 @@ EOF
 }
 
 @test "${tag}: restore ACLs, xattrs" {  # issue #1287
+    # Check if test needs to be skipped
+    touch "$BATS_TMPDIR/tmpfs_test"
+    run setfattr -n user.foo -v bar "$BATS_TMPDIR/tmpfs_test"
+    if [[ ("$status" != 0) && (-z $GITHUB_ACTIONS) ]]; then
+        skip 'xattrs unsupported by tmpfs'
+    fi
```

Comment:
You can do this more simply and idiomatically, something like:

```suggestion
    if    ! setfattr -n user.foo -v bar "$BATS_TMPDIR/tmpfs_test" \
       && [[ -z $GITHUB_ACTIONS ]]; then
        skip "xattrs unsupported in ${BATS_TMPDIR}"
    fi
```

If you did keep `run`, don't forget to `echo "$output"`.